back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -282,7 +282,7 @@ def idea_attributes(custom_form, user_can_moderate_project)
 
   def idea_simple_attributes(submittable_field_keys)
     simple_attributes = %i[location_description proposed_budget] & submittable_field_keys
-    simple_attributes.push(:publication_status, :project_id, :author_id)
+    simple_attributes.push(:publication_status, :project_id, :author_id, :anonymous)
```

Comment:
We should only allow this if the project/phase allows anonymous posts. Is this something we want to tackle here, or will it be part of a separate PR?

Reply:
Good point, will need your PR merged first, so will put it in a separate ticket